iceoryx2/src/port/subscriber.rs

```diff
@@ -136,15 +138,28 @@ impl<Service: service::Service, MessageType: Debug> Subscriber<Service, MessageT
 
         let dynamic_storage = Arc::clone(&service.state().dynamic_storage);
 
+        let buffer_size = match config.buffer_size {
+            Some(buffer_size) => {
+                if static_config.subscriber_max_buffer_size < buffer_size {
```

Comment:
Out of curiosity since I also saw this somewhere else. Do you prefer the `treshold <=> variable` notation? I haven't seen it in your PRs the last years.

Reply:
actually I prefer `<` over this `>` ... I think Walter E. Brown recommended it also in one of his talks.

Reply:
For me that order feels a bit unnatural. But one gets used to